packages/react/src/components/Tabs/Tabs.mdx

```diff
@@ -13,9 +13,14 @@ import { Tabs, TabList, Tab, TabPanels, TabPanel } from './Tabs';
 
 - [Overview](#overview)
   - [Line Tabs](#line-tabs)
-  - [Container Tabs](#container-tabs)
+  - [Contained Tabs](#contained-tabs)
+  - [Icon Tabs](#icon-tabs)
 - [Component API](#component-api)
-  - [Tab `renderContent`](#tab-rendercontent)
+  - [Tab - render content on click](#tab---render-content-on-click)
+- [V11](#v11)
+  - [Tabs composition](#tabs-composition)
+  - [Various updates](#various-updates)
+  - [Max width](#max-width)
```

Comment:
Thanks for running doctoc!

Reply:
there's an automated way to do this? 🤣 I wrote it hahahah

Reply:
haha, yes! Run `yarn doctoc` in the root